engine/src/main/java/org/hibernate/search/engine/backend/types/dsl/StringIndexFieldTypeOptionsStep.java

```diff
@@ -65,4 +66,10 @@
 	 */
 	S termVector(TermVector termVector);
 
+	/**
+	 * @param highlightable Whether highlighting is supported and if so which highlighter types can be applied.
+	 * @return {@code this}, for method chaining.
+	 * @see Highlightable
+	 */
+	S highlightable(Highlightable... highlightable);
```

Comment:
As discussed elsewhere, you might want to pass a collection here, to avoid confusion where users call the method without any argument.

```suggestion
	/**
	 * @param highlightable Whether highlighting is supported and if so which highlighter types can be applied.
	 * Pass {@code Collections.singleton(Highlightable.NO) to disable highlighting.
	 * @return {@code this}, for method chaining.
	 * @see Highlightable
	 */
	S highlightable(Collection<Highlightable> highlightable);
```

Reply:
I've made the same change in `PropertyMappingFullTextFieldOptionsStep` too.